Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Carousel/CarouselPage.xaml

```diff
@@ -8,8 +8,9 @@
       mc:Ignorable="d">
 
     <Grid Background="{StaticResource Brush-Grey-05}">
-        <Border Margin="0">
-            <controls:Carousel x:Name="CarouselControl"
+   
+            <Border Margin="0">
```

Comment:
Agree with this, even though this all cosmetical, the code would look a bit cleaner if the extra tab and space is removed :)